rosdep/python.yaml

```
@@ -8303,6 +8303,13 @@ python3-pyqt5.qtwebengine:
   openembedded: ['${PYTHON_PN}-pyqt5@meta-qt5']
   opensuse: [python3-qtwebengine-qt5]
   ubuntu: [python3-pyqt5.qtwebengine]
+python3-pyqt6.qtmultimedia:
+  debian:
+    bullseye: null
+    '*': [python3-pyqt6.qtmultimedia]
+  ubuntu:
+    '*': null
+    'noble': [python3-pyqt6.qtmultimedia]
```

We typically assume that if a distribution has it, then newer ones will as well.  So we should reverse the order here:

```suggestion
  debian:
    '*': [python3-pyqt6.qtmultimedia]
    bullseye: null
  ubuntu:
    '*': [python3-pyqt6.qtmultimedia]
    focal: null
    jammy: null
```